config/management/genesis/src/config_builder.rs

```diff
@@ -141,16 +141,27 @@ impl<T: AsRef<Path>> ValidatorBuilder<T> {
             )
             .unwrap();
 
-        validator_network.identity = Identity::from_storage(
-            libra_global_constants::VALIDATOR_NETWORK_KEY.into(),
-            libra_global_constants::OWNER_ACCOUNT.into(),
-            self.secure_backend(&local_ns, "validator"),
-        );
-        fullnode_network.identity = Identity::from_storage(
-            libra_global_constants::FULLNODE_NETWORK_KEY.into(),
-            libra_global_constants::OWNER_ACCOUNT.into(),
-            self.secure_backend(&local_ns, "full_node"),
-        );
+        if let Identity::FromStorage(identity) = validator_network.identity.clone() {
```

Comment:
why not just add a function called identity_from_storage in the NetworkConfig that does this so you don't end up with copypasta?